src/document-converter.ts

```diff
@@ -405,7 +416,9 @@ export class DocumentConverter {
         results.push({
           originalUrl: oldScriptUrl,
           convertedUrl: newScriptUrl,
-          convertedFilePath: getJsModuleConvertedFilePath(oldScriptUrl),
+          convertedFilePath:
+              replaceHtmlExtensionIfFound(oldScriptUrl as string) as
```

Comment:
(from offline) We're going to make a similar function to `getJsModuleConvertedFilePath` that performs a similar cast but is specifically documented as intended to produce the final file name with renaming applied; as opposed to `getJsModuleConvertedFilePath`, which produces a `ConvertedFilePath` without considering renaming. This works for this specific case because the only file we are currently renaming is 'polymer.html': `convertScriptUrl` is only applied to `OriginalDocumentUrl`s that were already referencing files that were already separate scripts so it won't result in a renamed file. If we intend to make it possible to rename any file (e.g. with a special comment) then we'll need to change the new function to read from the renaming information produced during scanning.

Reply:
Updated.